packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -512,3 +540,21 @@ class EventChannel {
     return controller.stream;
   }
 }
+
+/// A [BinaryMessenger] which uses [BinaryMessages] to send platform messages.
```

Comment:
I think it would be useful to say where those messages will go, i.e. the platform plugins.

Reply:
Done.